src/NTDll.Desktop/NTDll+OBJECT_ATTRIBUTES.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Diagnostics.CodeAnalysis;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="OBJECT_ATTRIBUTES"/> nested struct.
+    /// </content>
+    public static partial class NTDll
+    {
+        /// <summary>
+        /// The OBJECT_ATTRIBUTES structure specifies attributes that can be applied to objects or object handles by routines that create objects and/or return handles to objects.
+        /// </summary>
+        [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
+        public struct OBJECT_ATTRIBUTES
```

Comment:
If based on my comments in this struct you add pointer fields, please add `[OfferIntPtrPropertyAccessors]` here, and set the Custom Tool property on the file to `MSBuild:GenerateCodeFromAttributes`